apps/remix-ide/src/app/udapp/run-tab.js

```diff
@@ -44,6 +45,31 @@ export class RunTab extends LibraryPlugin {
     this.compilersArtefacts = compilersArtefacts
     this.networkModule = networkModule
     this.fileProvider = fileProvider
+    this.createVMAccount = this.pluginUDapp.createVMAccount
```

Comment:
if PluginUdapp is not used elsewhere, I think just removing it would be better

Reply:
Are you saying createVMAccount and the other methods are not used?

Reply:
Are you saying createVMAccount and the others methods are not used? or just that the class is not needed.

Reply:
no, just the `PluginUdapp` (the class is not needed)